target/classes/avro/AnotherUser.avsc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do not commit target folder to Git. Delete it and add target folder to .gitignore.